src/bundles/SellerRegistration/components/DomainSelector/DomainSelector.js

```diff
@@ -55,27 +55,15 @@ class DomainSelector extends BaseForm {
             )}
 
             {domains.map(({ key, label, services }, i) => {
-              let additionalProps = {}
```

Comment:
Nope shouldn't be needed anymore, this was merely in place so that `ErrorBox` would link to the top level domains.